xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -195,14 +264,27 @@
         <Parameter Name="serialNumber" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="issuerName">To be added.</param>
-        <param name="generator">To be added.</param>
-        <param name="notBefore">To be added.</param>
-        <param name="notAfter">To be added.</param>
-        <param name="serialNumber">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="issuerName">The X500DistinguishedName for the Issuer.</param>
+        <param name="generator">An <see cref="T:System.Security.Cryptography.X509Certificates.X509SignatureGenerator"/> object representing the issuing certificate authority.</param>
+        <param name="notBefore">The oldest date and time when this certificate is considered valid. Typically <see cref="P:System.DateTimeOffset.UtcNow"/>, plus or minus a few seconds.</param>
+        <param name="notAfter">The date and time when this certificate is no longer considered valid.</param>
+        <param name="serialNumber">The serial number to use for the new certificate. This value should be unique per issuer. The value is interpreted as an unsigned (big) integer in big-endian byte ordering.</param>
+        <summary>Signs the current certificate request to create a chain-signed or self-signed certificate.</summary>
+        <returns>The ASN.1 DER-encoded certificate, suitable to be passed to the <see cref="M:System.Security.Cryptography.X509Certificates.X509Certificate2.#ctor(System.Byte[])"/> constructor.</returns>
```

Comment:
I think this `<returns>` didn't get updated as the API changed shape.  Since it returns an X509Certificate2 object instead of the bytes for one the sentence doesn't make sense anymore.  Probably should be copied from the previous overload.